pages/common/cargo-update.md

```diff
@@ -0,0 +1,20 @@
+# cargo update
+
+> Check a local package and all of its dependencies for errors.
+> More information: <https://doc.rust-lang.org/cargo/commands/cargo-update.html>.
+
+- Update dependencies in the `Cargo.lock` file to the latest version:
```

Comment:
Should we also mention `minor` patch version here.
IMO, it does not do a major patch verison bump.

Reply:
good point!

Reply:
should be done now